controllers/job_controller.js

```diff
@@ -1,82 +1,80 @@
+const jobs = require("../models/jobs.js");
 const Jobs = require("../models/jobs.js");
 const Users = require("../models/users.js");
 
 const controller = {
   async showJobPage(req, res) {
     try {
       const jobId = req.params.job_id;
-      const job = await Jobs.findById(jobId).populate("author_id");
+      const job = await Jobs.findByIdAndUpdate(jobId, { $pull: { photos: null, skills: null } }).populate("author_id");
       res.render("./jobs/index.ejs", { job });
     } catch (error) {
       console.log(error.message);
     }
   },
-  // //Method GET
-  // newProjectForm(req, res) {
-  //   res.render("./projects/project_new.ejs");
-  // },
+  //Method GET
+  new(req, res) {
```

Comment:
Why suddenly your naming convention change?

you didn’t name the top function as show